src/main/kotlin/com/terraformation/backend/tracking/db/ObservationResultsStore.kt

```diff
@@ -827,21 +828,42 @@ class ObservationResultsStore(private val dslContext: DSLContext) {
       )
 
   private fun plantingSubzoneSpeciesMultiset(): Field<List<ObservationSpeciesResultsModel>> {
+    listOf(
+            PLOT_T0_DENSITIES,
+            MONITORING_PLOTS,
+            MONITORING_PLOT_HISTORIES,
+            PLANTING_SUBZONES,
+            PLANTING_SUBZONE_HISTORIES,
+            PLANTING_ZONES,
+            PLANTING_ZONE_HISTORIES,
+            PLANTING_SITES,
+            PLANTING_SITE_HISTORIES,
+            OBSERVATION_PLOTS,
+            OBSERVATIONS,
+        )
+        .forEach { println("${it.name}\n${dslContext.fetch(it)}") }
```

Comment:
Remove this

Reply:
Ah thanks. I had planned on using it as part of testing the changes for the temp t0 data, but then decided to put that in a separate PR